src/app/ApplicationTemplate.Shared/CoreStartup.cs

```diff
@@ -58,6 +59,15 @@ protected override async Task StartServices(IServiceProvider services, bool isFi
 
 				services.GetRequiredService<DiagnosticsCountersService>().Start();
 
+//-:cnd:noEmit
+#if __ANDROID__ || __IOS__
```

Comment:
Wouldn't it be cleaner if this condition was in the method rather than outside of it?